server/schemas/request.graphql

```diff
@@ -25,6 +25,8 @@ type RequestItem {
   version: String
   ref: String
   item: VersionedItem
+  schema: Schema
+  model: Model
```

Comment:
```suggestion
```

Add these fields and resolvers to the Item type instead of adding it here.

Reply:
Do you mean the versioned Item?

Reply:
No, `Item` type